monitoring-server/cmake/Project.cmake

```diff
@@ -6,6 +6,7 @@ target_sources(Core
         include/core/comms/I2C.h
         include/core/comms/Packet.h
         include/core/events/MessageDelivery.h
+        include/core/events/StopServer.h
```

Comment:
Please refer to the first comment in "monit/core: Create Device class definition" commit for similar question.

Reply:
Same as above